test/button/button-subtle.html

```diff
@@ -0,0 +1,101 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+		<title>subtle button tests</title>
+
+		<script src="../../../@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+		<script src="../../../mocha/mocha.js"></script>
+		<script src="../../../chai/chai.js"></script>
+		<script src="../../../@polymer/test-fixture/test-fixture.js"></script>
+		<script src="../../../wct-mocha/wct-mocha.js"></script>
+
+		<script type="module" src="../../components/button/button-subtle.js"></script>
+	</head>
+	<body>
+		<test-fixture id="basic">
+			<template>
+				<d2l-button-subtle text="Subtle Button"></d2l-button-subtle>
+			</template>
+		</test-fixture>
+		<test-fixture id="with-icon">
+			<template>
+				<d2l-button-subtle text="Subtle Button with Icon" icon="d2l-tier1:gear"></d2l-button-subtle>
+			</template>
+		</test-fixture>
+		<test-fixture id="disabled">
+			<template>
+				<d2l-button-subtle disabled text="Disabled Subtle Button"></d2l-button-subtle>
+			</template>
+		</test-fixture>
+
+		<script type="module">
+			import { runAxe } from '../../test-helpers/a11y-helper.js';
+
+			suite('subtle button', function() {
+				let button;
+				suite('basic', function() {
+					setup(function(done) {
+						button = fixture('basic');
+						requestAnimationFrame(function() {
+							done();
+						});
+					});
+					test('should execute onclick when clicked', function(done) {
```

Comment:
This was the only old test I brought over.  Every other one seemed like it could be tested by perceptual diff, but interested in your thoughts: https://github.com/BrightspaceUI/button/blob/master/test/button-subtle.html

Reply:
I agree that everything else could be tested with perceptual diff